unpackaged/config/recover_parent_funding_program_analytics/package.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Package xmlns="http://soap.sforce.com/2006/04/metadata">
+    <types>
```

Comment:
I believe you need a `<fullName>` element here with the name of the OFM managed package (_not_ the namespace) - see the `package.xml` in PMM's example.